docs/filters-and-examples.md

```diff
@@ -0,0 +1,349 @@
+# Filters
+
+The Search plugin comes with a set of predefined search filters that allow you to
+easily create the search results you need. Use:
+
+----------
+
+`Value` to limit results to exact matches
+<details>
```

Comment:
The details are collapsed by default and I don't like the fact that an additional click is required to view the code e.g.

Reply:
since most of these code snippets are only one liners I can actually remove that collapsible element.